eth/test/utils/TestWorld.ts

```diff
@@ -63,11 +63,6 @@ export async function initializeWorld({
 }: InitializeWorldArgs): Promise<World> {
   const [deployer, user1, user2] = await hre.ethers.getSigners();
 
-  // The tests assume that things get mined right away
-  // TODO(#912): This means the tests are wildly fragile and probably need to be rewritten
```

Comment:
glad we're getting rid of this code

Reply:
I removed these, but the **issue** doesn't go away. Ideally, we'd get hardhat to support interval mining within it's test helpers. I'm going to talk to Franco